admin/lib/solidus_admin/configuration.rb

```diff
@@ -23,16 +23,16 @@ class Configuration < Spree::Preferences::Configuration
     #
     # @see https://tailwindcss.com/docs/configuration#content
     preference :tailwind_content, :array, default: [
-      SolidusAdmin::Engine.root.join("public/*.html"),
       SolidusAdmin::Engine.root.join("app/helpers/**/*.rb"),
       SolidusAdmin::Engine.root.join("app/assets/javascripts/**/*.js"),
-      SolidusAdmin::Engine.root.join("app/views/**/*.{erb,haml,html,slim}"),
-      SolidusAdmin::Engine.root.join("app/components/**/*.rb"),
+      SolidusAdmin::Engine.root.join("app/views/**/*.erb"),
+      SolidusAdmin::Engine.root.join("app/components/**/*.{rb,erb,js}"),
+
       Rails.root.join("public/solidus_admin/*.html"),
       Rails.root.join("app/helpers/solidus_admin/**/*.rb"),
       Rails.root.join("app/assets/javascripts/solidus_admin/**/*.js"),
       Rails.root.join("app/views/solidus_admin/**/*.{erb,haml,html,slim}"),
-      Rails.root.join("app/components/solidus_admin/**/*.rb")
+      Rails.root.join("app/components/solidus_admin/**/*.{rb,erb,haml,html,slim,js}"),
```

Comment:
We also need to update the template copied to the host application: https://github.com/solidusio/solidus/blob/4bd4ff5bcbc117dfeba11bf5e437453073218963/admin/lib/generators/solidus_admin/install/templates/config/initializers/solidus_admin.rb#L16

Super-neat: remove last `,` :slightly_smiling_face:

Reply:
re `,`: I removed it just because you asked 😊 

I use them to reduce noise in diffs and syntax errors when I add another line and don't notice there's a missing comma, but I'm curious about your perspective on this.

Reply:
I don't have a strong opinion here, so feel free to re-add it :slightly_smiling_face: I feel it's nice to remove it because it's not needed, but your points also make total sense!